src/backend/distributed/planner/insert_select_planner.c

```diff
@@ -96,6 +96,17 @@ static List * AddInsertSelectCasts(List *insertTargetList, List *selectTargetLis
 								   Oid targetRelationId);
 static Expr * CastExpr(Expr *expr, Oid sourceType, Oid targetType, Oid targetCollation,
 					   int targetTypeMod);
+static Oid GetNextvalReturnTypeCatalog(void);
+static void append_casted_entry(TargetEntry *insertEntry, TargetEntry *selectEntry,
+								Oid castFromType, Oid targetType, Oid collation, int32
+								typmod,
+								int targetEntryIndex,
+								List **projectedEntries, List **nonProjectedEntries);
+static void set_target_entry_name(TargetEntry *tle, const char *format, int index);
+static void reset_target_entry_resno(List *targetList);
+static void process_entry_pair(TargetEntry *insertEntry, TargetEntry *selectEntry,
+							   Form_pg_attribute attr, int targetEntryIndex,
+							   List **projectedEntries, List **nonProjectedEntries);
```

Comment:
As in `GetNextvalReturnTypeCatalog()`, we prefer CamelCase in code-base over snake-case, except the functions that we copy from Postgres, so it would be better if we rename these functions before merging.

Reply:
converted